src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -65,6 +65,13 @@ object ErrorAggregator {
     .add[Int]("SLOW_SCRIPT_PAGE_COUNT")
     .build
 
+  private val thresholdHistograms = Map(
+    "INPUT_EVENT_RESPONSE_COALESCED_MS" -> (List("main", "content"), List(150, 250, 2500)),
+    "GHOST_WINDOWS" -> (List("main", "content"), List(1)),
+    "GC_MAX_PAUSE_MS" -> (List("main", "content"), List(150, 250, 2500)),
```

Comment:
We changed it to be both `opt-out`, as well as have a larger `range` and `bucket-count` which now matches `cycle_collector_max_pause`.